tests/integrationv2/utils.py

```diff
@@ -62,7 +46,8 @@ def invalid_test_parameters(*args, **kwargs):
     curve = kwargs.get('curve')
     signature = kwargs.get('signature')
 
-    providers = [provider_ for provider_ in [provider, other_provider] if provider_]
+    providers = [provider_ for provider_ in [
+        provider, other_provider] if provider_]
```

Comment:
This formatting is kind of hard to read with the line break (although it was already kind of hard to read). Maybe this?

```
providers = [
    provider_ for provider_ in [
        provider, other_provider
    ] if provider_
]
```

Reply:
This was formatted by autopep8, to pass the GHA.

Reply:
https://github.com/hhatto/autopep8/issues/460